src/Toplo-Widget-Menu/AnObsoleteToChoiceMenu.class.st

```diff
@@ -0,0 +1,22 @@
+Class {
+	#name : #AnObsoleteToChoiceMenu,
+	#superclass : #ToMenu,
+	#traits : 'TToCheckableGroupHolder',
+	#classTraits : 'TToCheckableGroupHolder classTrait',
+	#instVars : [
+		'group',
+		'data',
+		'dataToTextConverter',
+		'defaultLabel',
+		'defaultSelectedIndex'
+	],
+	#category : #'Toplo-Widget-Menu-Choice menu deprecated'
+}
```

Comment:
I didn't see this addition of AnObsoleteToChoiceMenu on my Iceberg window...
And I should had replaced the reference to ToChoiceMenu that breaks the 26 tests